pkg/set/uint.go

```diff
@@ -18,27 +25,37 @@ func (e ErrConflict) Error() string {
 // UintSet is a set containing uint
 type UintSet struct {
 	sync.RWMutex
-	m map[uint]struct{}
+	root string
 }
 
 // NewUint creates a new set for uint
-func NewUint() *UintSet {
+// path if the directory where to store the set on disk
+// the directory pointed by path must exists already
+func NewUint(path string) *UintSet {
```

Comment:
If this set is created for a location under `/tmp/ports` for example, then we won't need to check for the first-boot status. Since the /tmp folder is gonna be free after new boot anyway. The only concern is free space limitation under /tmp but I think these port files are extremely small anyway. 

Other than that, i think code is okay

Reply:
Right, indeed. that would solve the problem...
And actually we could use the same trick there too then: https://github.com/threefoldtech/zos/blob/04ffb3459581917e76eb24cde639e547821bd0e8/pkg/network/ndmz/ndmz.go#L60-L93

What about I implement a package which purpose is to return path that are usable for different use case.
- persistent forever
- persistent between daemon restart

Reply:
Sounds like a good idea to me actually. We just need to make sure either to mount a good enough tmpfs size  for run-time data (/tmp) to be able to hold all temporary data.